Week5.Rmd

```diff
@@ -0,0 +1,222 @@
+  group_by(X_state, checkup1) %>%
+  mutate(checkup_recently = ifelse(checkup1 == "Within past year" | checkup1 == "Within past 2 years", "yes", "no")) %>%
+  group_by(X_state, checkup_recently) %>%
+  summarise(sum_recent_exam = sum(count))
+
+percentage_exam <- exams_by_state %>%
+  group_by(X_state) %>%
+  mutate(percent_exam = prop.table(sum_recent_exam) * 100) %>%
+  filter(checkup_recently == "yes")
+
+ungroup(percentage_exam) %>%
+  mutate(X_state = fct_reorder(X_state, percent_exam)) %>%
+  ggplot( aes(x = X_state, y = percent_exam, fill = percent_exam)) + 
+    geom_bar(stat = "identity", width = 0.5) + 
+    coord_flip() +
+    theme_minimal() +
+    labs(title = "Population Percentage with Wellness Exam within Past 2 Years by State",
+         fill = "Percentage",
+         x = "Percentage",
+         y = "State")
+```
+
+There does appear to be some variation in the percentage of the population of each state that has received a wellness exam within the last 2 years. The highest percentage which is in Massachusetts is almost 93% while the lowest which is in Alaska is just below 75%.
+
+```{r}
+ggplot(percentage_exam, aes(x = percent_exam)) +
+  geom_boxplot() +
+  geom_dotplot(stackdir = 'center', binwidth = 1, method = "histodot", dotsize = 0.5, fill = "grey") +
+  labs(title = "Distribution of Population Percentages Above",
+       x = "Percentage") +
+  theme(axis.title.y =  element_blank(),
+        axis.text.y = element_blank(),
+        axis.ticks.y = element_blank())
+```
+
+The median percentage of the population of a given state that has had a wellness exam within 2 years is about 86% and there are no outliers.
+
+```{r}  
+insured_by_state <- insurance_exams %>% 
+  group_by(X_state, hlthpln1) %>%
+  summarise(num_insured = sum(count))
+
+percentage_insured <- insured_by_state %>%
+  group_by(X_state) %>%
+  mutate(percent_insured = prop.table(num_insured) * 100) %>%
+  filter(hlthpln1 == "Yes")
+
+yes_percentages <- merge(percentage_exam, percentage_insured, by = "X_state")
+
+ggplot(yes_percentages, aes(x = percent_insured, y = percent_exam, label = X_state)) + geom_point(color = "darkgrey") +   geom_text_repel(aes(label = X_state),
+                 point.padding = NA,
+                 size = 2.5) +
+  labs(title = "Percentage of Population with Recent Checkup vs Percent with Insurance",
+       x = "Percent of State Population with Health Insurance",
+       y = "Percent of State Population with Checkup in Past 2 Years")
+```
+
+There is a positive correlation between the percentage of the population of a given state with health insurance and the percentage of that population that has had a checkup within the past 2 years. However, the scatter along the y-axis suggests that there are regional differences in the percentage of the population that has had a wellness exam recently. For example, Alaska has much lower rates than other states with comparable levels of health insurance coverage.
```

Comment:
There is a mention of a positive correlation here, but no R value is given in the chart or in the graph.